README.md

```diff
@@ -41,6 +42,9 @@ See the [AutoGluon Website](http://autogluon.mxnet.io/index.html) for instructio
 - [Accurate image classification in 3 lines of code with AutoGluon](https://medium.com/@zhanghang0704/image-classification-on-kaggle-using-autogluon-fc896e74d7e8) (*Medium*, Feb 2020)
 - [AutoGluon overview & example applications](https://towardsdatascience.com/autogluon-deep-learning-automl-5cdb4e2388ec?source=friends_link&sk=e3d17d06880ac714e47f07f39178fdf2) (*Towards Data Science*, Dec 2019)
 
+### Tutorial Notebooks
```

Comment:
This section's title may confuse users into thinking these are all the available tutorials.  Consider renaming to something like:  "Other Resources", "More Notebooks", "External Notebooks", "Supplementary Notebooks", ...

Reply:
Renamed to "Supplementary Notebooks"